pymc/smc/smc.py

```diff
@@ -41,9 +41,9 @@
 
 
 class SMC_KERNEL(ABC):
-    """Base class for the Sequential Monte Carlo kernels
+    """Base class for the Sequential Monte Carlo kernels.
```

Comment:
I need to review this after looking at the preview, I am not sure what is going on.

Reply:
👍